### ISSUE TITLE
Cannot read property 'deps' of null

### ISSUE THREAD

**Issue by Mattchewone** [vuejs/vue-test-utils#550]
### Version
1.0.0-beta.14

### Reproduction link
[https://codesandbox.io/s/1pm8pzxm4](https://codesandbox.io/s/1pm8pzxm4)

### Steps to reproduce
Open the test tab to see the failing test.

### What is expected?
Using `findAll` with `.at()` to trigger a click

### What is actually happening?
Test is failing with "Cannot read property 'deps' of null" message

---
I am storing open/close states in parent component and testing by triggering clicks in the scopedSlot component.

**Comment by Mattchewone** [comment]
Also not sure why in the `codesandbox` example these aren't printed in the console, but on my local version my terminal is filled with these:
```
[Vue warn]: Unknown custom element: <v-card-title>
```

I can't see what is wrong with the way I have added Vuetify to the localVue and Vue instance. So any insights would be greatly appreciated.

**Comment by Mattchewone** [comment]
@eddyerburgh Could you point me in the right direction as to where I could look into this so I could perhaps create a PR for a fix. Assuming this is a bug and not an issue with my code..

Thanks

**Comment by eddyerburgh** [comment]
The issue will be in either this file—https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/src/set-watchers-to-sync.js, or this file—https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/src/order-watchers.js

**Comment by Mattchewone** [comment]
Thanks @eddyerburgh Is there a guide on getting the test utils running locally. I finally managed to get it to installed but getting lots of flow check errors when running the test suite. Wanted to get it to run locally before I looked into those files. 
Maybe I am missing something from the setup, but I cloned and then installed, and it kept failing on the postinstall with a `lerna` failure. So I installed some deps to get passed that, but then running `npm test` just fails on `flow check` - `Module not found: ...`
